elements/bulbs-video/bulbs-video-cms.js

```diff
@@ -0,0 +1,17 @@
+import {
+  registerElement,
+  EmbededCMSElement,
+} from 'bulbs-elements/register';
+
+class EmbeddedBulbsVideo extends EmbededCMSElement {
+  get embedContentPreview () {
+    return `
+      <h1 style='text-align: center; font-family: "Droid Serif"'>
+        <i class='fa fa-puzzle-piece'></i>
+        Embedded BulbsVideo
```

Comment:
Wonder if we should nip this early so we don't have the same problem as the polls element - maybe we have this embedded CMS element make a request to the Video Hub to get the video title and display it?